src/main/java/org/zalando/nakadi/service/timeline/TimelineService.java

```diff
@@ -129,12 +148,26 @@ public Timeline getTimeline(final EventTypeBase eventType) throws TimelineExcept
         }
     }
 
+    public Timeline getFakeTimeline(final EventType eventType) {
+        try {
+            final Storage storage = storageDbRepository.getStorage(DEFAULT_STORAGE)
+                    .orElseThrow(() -> new UnableProcessException("No default storage defined"));
```

Comment:
I think we agreed that method should define which exceptions it throws. 
Also, I'm not sure if UnableProcessException is a right exception to throw here, for me it's more like InconsistentStateException as we could not find default storage which should always be there.